Signal/src/ViewControllers/ContactViewController.swift

```diff
@@ -573,7 +573,7 @@ class ContactViewController: OWSViewController, ContactShareViewHelperDelegate {
         Logger.info("\(self.logTag) \(#function)")
 
         guard let url = NSURL(string: "tel:\(phoneNumber.phoneNumber)") else {
-            owsFail("\(ContactViewController.logTag) could not open phone number.")
+            owsFail("\(ContactViewController.logTag()) could not open phone number.")
```

Comment:
2. Ensure all usages of class `logTag` property are invoked.

Reply:
hmmm... isn't this obviated by:

```
3. Convert class method to class property.
```

Reply:
Unfortunately, no.  I haven't looked into why yet.